dashboard/key-visualizer.md

```diff
@@ -0,0 +1,181 @@
+---
+title: 流量可视化页面
+category: how-to
+aliases: ['/docs-cn/dev/how-to/monitor/key-visualizer/','/docs-cn/dev/key-visualizer-monitoring-tool/']
+---
+
+# 流量可视化页面
+
+流量可视化页面 (Key Visualizer) 可用于分析 TiDB 集群的使用模式和排查流量热点。该页面可视化地呈现了 TiDB 集群一段时间的流量情况。
+
+## 访问页面
+
+可以通过以下两种方法访问 Key Visualizer 流量可视化页面：
+
+* 登录 TiDB Dashboard 后，点击左侧导航条的 **Key Visualizer**（流量可视化）：
+
+![访问](/media/dashboard/key-visualizer/access.png)
+
+* 在浏览器中访问 `http://127.0.0.1:2379/dashboard/#/keyviz`（将 `127.0.0.1:2379` 替换为实际的 PD 地址和端口）。
+
+> **注意：**
+>
+> 流量可视化功能默认关闭，在初次使用时需要手动打开。请依据界面指引来开启该功能，或参见本文档[设置](#设置)部分。
+
+## 界面示例
+
+流量可视化页面示例如下：
+
+![Key Visualizer 示例图](/media/dashboard/key-visualizer/overview.png)
+
+从以上流量可视化界面可以观察到以下信息：
+
++ 一个大型热力图，显示整体访问流量随时间的变化情况。
++ 热力图某个坐标的详细信息。
++ 左侧为表、索引等标识信息。
+
+## 基本概念
+
+本节介绍流量可视化涉及的一些基本概念。
+
+### Region
+
+在 TiDB 集群中，数据以分布式的方式存储在所有的 TiKV 实例中。TiKV 在逻辑上是一个巨大且有序的 KV Map。整个 Key-Value 空间分成很多 Region，每一个 Region 是一系列连续的 Key。
+
+> **注意：**
+>
+> 关于 Region 的详细介绍，请参考[三篇文章了解 TiDB 技术内幕 - 说存储](https://pingcap.com/blog-cn/tidb-internal-1/#region)
+
+### 热点
+
+在使用 TiDB 的过程中，热点是一个典型的现象，它表现为大量的流量都在读写一小块数据。由于连续的数据往往由同一个 TiKV 实例处理，因此热点对应的 TiKV 实例的性能就成为了整个业务的性能瓶颈。常见的热点场景有使用自增主键连续写入相邻数据导致的写入表数据热点、时间索引下写入相邻时间数据导致的写入表索引热点等。
+
+> **注意：**
+>
+> 热点问题详情请参阅 [TiDB 热点问题详解](https://asktug.com/t/topic/358)。
+
+### 热力图
+
+热力图是流量可视化页面的核心，它显示了一个指标随时间的变化。热力图的横轴 X 是时间，纵轴 Y 则是按 Key 排序的连续 Region，横跨 TiDB 集群上所有数据库和数据表。颜色越暗 (cold) 表示该区域的 Region 在这个时间段上读写流量较低，颜色越亮 (hot) 表示读写流量越高，即越热。
+
+### Region 压缩
+
+一个 TiDB 集群中，Region 的数量可能多达数十万。在屏幕上是难以显示这么多 Region 信息的。因此，在一张热力图中，Region 会被压缩到约 1500 个连续范围，每个范围称为 Bucket。在一个热力图上，热的实例更需要关注，因此 Region 压缩总是倾向于将流量较小的大量 Region 压缩为一个 Bucket，而尽量让高流量的 Region 独立成 Bucket。
+
+## 使用介绍
+
+本节介绍如何使用流量可视化页面。
+
+### 设置
+
+首次使用流量可视化页面需要先通过**设置**页面手动开启此功能。参考页面指引，点击 **Open Settings**（打开设置）即可打开设置页面：
```

Comment:
Chinese should be put first considering this is a Chinese doc?

Reply:
But the UI displays buttons in English. It would be weird to call a button name that's not in the UI.

Reply:
In actual UI it is displayed in Chinese.